source/framework/core/src/TRestHits.cxx

```diff
@@ -842,20 +851,19 @@ Double_t TRestHits::GetDistanceToNode(Int_t n) {
     return distance;
 }
 
-Int_t TRestHits::GetMostEnergeticHitInRange(Int_t n, Int_t m) {
-    Int_t maxEn = 0;
+Int_t TRestHits::GetMostEnergeticHitInRange(Int_t n, Int_t m) const {
+    double maxEnergy = 0;
```

Comment:
This can also change the pipeline behaviour but it is indeed a bug, since energy should be `double`

Reply:
yes, this is the only thing I can think of... but I don't think this method is used right? hopefully its not used anywhere since it was bugged.

Reply:
```
jgalan@sultan:~/rest-framework/source$ grep -rnw . -e "GetMostEnergeticHitInRange"
./libraries/track/src/TRestTrackBlobAnalysisProcess.cxx:215:        Int_t hit1 = hits->GetMostEnergeticHitInRange(0, nCheck);
./libraries/track/src/TRestTrackBlobAnalysisProcess.cxx:216:        Int_t hit2 = hits->GetMostEnergeticHitInRange(nHits - nCheck, nHits);
```